packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1890,6 +1890,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
 
     public process(messageArg: ISequencedDocumentMessage, local: boolean) {
         this.verifyNotClosed();
+        if (this.opsCurrentlyProcessing !== 0) {
```

Comment:
I do not remember why we decided to go with checking it in two places, but worth adding some comment on why we are doing